lib/pages/new_group/contacts_selection_view.dart

```diff
@@ -19,24 +19,24 @@ class ContactsSelectionView extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    return Padding(
-      padding: ContactsSelectionViewStyle.parentPadding,
-      child: Scaffold(
-        backgroundColor: LinagoraSysColors.material().onPrimary,
-        appBar: PreferredSize(
-          preferredSize: SearchableAppBarStyle.maxPreferredSize(context),
-          child: SearchableAppBar(
-            focusNode: controller.searchFocusNode,
-            title: controller.getTitle(context),
-            searchModeNotifier: controller.isSearchModeNotifier,
-            hintText: controller.getHintText(context),
-            textEditingController: controller.textEditingController,
-            openSearchBar: controller.openSearchBar,
-            closeSearchBar: controller.closeSearchBar,
-            isFullScreen: controller.isFullScreen,
-          ),
+    return Scaffold(
```

Comment:
This one is not used ?

Reply:
this one is used in creating new group chat view